src/GraphQL/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Antlr4" version="4.5-alpha002" targetFramework="net45" userInstalled="true" />
```

Comment:
Okay, makes sense. We don't do `il-merge` in our build chain, hence why I ran into this. But can change that. Will remove this from the PR.